src/ploomber/telemetry/telemetry.py

```diff
@@ -69,11 +72,14 @@ def is_online():
 
 # Will output if the code is within a container
 def is_docker():
-    cgroup = Path('/proc/self/cgroup')
-    docker_env = Path('/.dockerenv')
-    return (docker_env.exists() or cgroup.exists()
-            and any('docker' in line
-                    for line in cgroup.read_text().splitlines()))
+    try:
+        cgroup = Path('/proc/self/cgroup')
+        docker_env = Path('/.dockerenv')
+        return (docker_env.exists() or cgroup.exists()
+                and any('docker' in line
+                        for line in cgroup.read_text().splitlines()))
+    except FileNotFoundError:
+        return False
```

Comment:
will this ever raise FileNotFoundError? `Path(...)` will always work

Reply:
It can, it's just the wrong error. Look here: https://docs.python.org/3/library/pathlib.html (look for exist) and here https://docs.python.org/3/library/exceptions.html#OSError

It's been fixed in 3.8 but in 3.7/6 it'll still raise an OSError